Sources/VLCPlaybackController.m

```diff
@@ -375,9 +375,7 @@ - (void)_savePlaybackState
         APLog(@"failed to save current media state - file removed?");
     }
 }
-#endif
 
-#if TARGET_OS_IOS
```

Comment:
I don't understand this VLCPlaybackController change in this commit.
I believe it should be a separate commit.

Reply:
could be. I just saw it and removed it